src/lib/guard/guard.factory.test.ts

```diff
@@ -132,15 +142,43 @@ describe('Guard Factory', () => {
     );
   });
 
+  it('should keep underscore in path and name', async () => {
```

Comment:
```suggestion
  it("should keep underscores guad's path", async () => {
```

I'm assuming that the 'name' is `BarGuard`

Reply:
I've followed this for all tests and refactored the existing ones.